sites/profoodworld/redirect-handler.js

```diff
@@ -0,0 +1,19 @@
+const digital = async (from) => {
+  const pattern = /\/sites\/default\/files\/digital_edition\/(.*)/;
+  const matches = pattern.exec(from);
+  if (matches && matches[1]) {
+    return { to: `https://digitaleditions.profoodworld.com/${matches[1]}` };
+  }
+  return null;
+};
+
+const s3 = async (from) => {
+  const pattern = /\/sites\/default\/files\/(.*)/;
+  const matches = pattern.exec(from);
+  if (matches && matches[1]) {
+    return { to: `https://s3.us-east-2.amazonaws.com/pmg-production/Migrated+-+DO+NOT+USE/PFW/${matches[1]}` };
+  }
+  return null;
+};
+
+module.exports = async ({ from }) => Promise.all([digital(from), s3(from)]).then(l => l[0]);
```

Comment:
This is only ever going to return the `digital` value

Reply:
From what I read and tested, node will not set a key as null. So the first one in the list would be the first match.

Reply:
Those aren't keys... it's an array of promises...
Regardless, object keys and be set to _anything_